sysutils/bash-completion/files/patch-xpdf-support-compressed-pdf.diff

```diff
@@ -3,19 +3,16 @@ Subject: xpdf in Debian also supports compressed PDFs
 Origin: vendor, https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=702498
 Bug-Debian: https://bugs.debian.org/702498
```

Comment:
Same here, Debian has not re-rolled this patch for 2.12 yet.

https://salsa.debian.org/debian/bash-completion/-/blob/master/debian/patches/06-xpdf_support_compressed_pdf.patch?ref_type=heads